### DISCUSSION TITLE
couldn't send discord message Only absolute URLs are supported

### DISCUSSION THREAD

**Discussion by subasically** [jef/streetmerchant#1207] [Help]
Error:
![image](https://user-images.githubusercontent.com/26078024/101262961-a788ef00-3707-11eb-865f-86982fe7b691.png)

.env file
```
DISCORD_NOTIFY_GROUP="<@&784873022154342411>"
DISCORD_WEB_HOOK="https://discord.com/api/webhooks/##############/######-######################################"
```

**Comment by gigi2006** [comment]
Maybe better that you delete the Keys.

**Comment by jef** [comment]
I've recently do some work on this (#1209) but that PR won't solve this issue...

Unfortunately I am unable to duplicate as I've tested this in different situations, but your webhook url looks good.

**Comment by subasically** [comment]
Yeah I tested the webhook on another app(Plex) and it works fine. I've been looking at it for hours and I can't figure out what I am doing wrong.
